pkg/model/components/kubescheduler.go

```diff
@@ -39,8 +39,8 @@ func (b *KubeSchedulerOptionsBuilder) BuildOptions(o interface{}) error {
 
 	config := clusterSpec.KubeScheduler
 
-	if config.PolicyConfigMap != "" && b.IsKubernetesLT("v1.7.0") {
-		return fmt.Errorf("policyConfigMap is only supported in Kubernetes 1.7.0 or later")
+	if config.UsePolicyConfigMap != nil && b.IsKubernetesLT("v1.7.0") {
```

Comment:
Should this be Kubernetes version greater than?

Reply:
IsKubernetesGTE *

Reply:
I'm not sure I under your question right. This policyConfigMap feature was added in Kubernetes 1.7.0, and so I added a check that if the Kubernetes version is under that it will throw an error.

Reply:
You are correct, it's getting late for me :P